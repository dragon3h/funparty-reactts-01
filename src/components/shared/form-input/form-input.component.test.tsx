import React from 'react';
import * as renderer from 'react-test-renderer';

import FormInput from './form-input.component';

it('FormInput for email component renders correctly', () => {
  const formInput = renderer
      .create(<FormInput
      attrType={'email'}
      handleChange={jest.fn()}
      value={''}
      name={'Email'}
      label={'Email'}
      required={true}/>)
      .toJSON();
  
  expect(formInput).toMatchSnapshot();
});