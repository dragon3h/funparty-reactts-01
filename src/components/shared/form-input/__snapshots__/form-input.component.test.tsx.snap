// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput for email component renders correctly 1`] = `
<div
  className="input-group"
>
  <input
    className="form-input"
    name="Email"
    onChange={[MockFunction]}
    required={true}
    type="email"
    value=""
  />
  <label
    className=" form-input__label"
  >
    Email
  </label>
</div>
`;
